app/views/guests/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Guest: <%= @guest.name %></h1>
+
+<h3>Nights: <%= @guest.nights %></h3>
+
+<h3>Previous Rooms:</h3>
+<% @guest.rooms.each do |room| %>
+    <%= "Suite Name: #{room.suite}" %>
+    <ul>
+        <li><%= "Suite Hotel: #{room.hotel.name}" %></li>
+        <li><%= "Suite Rate: #{room.rate}" %></li>
+    </ul>
+<% end %>
+
+<%= form_with url: "/guests/#{@guest.id}", method: :get, local: true do |f| %>
```

Comment:
This is where, instead, you could route to "/guests/:id/rooms" and make it a POST, which could route to `guest_rooms#create`